docs/source/contributors/index.md

```diff
@@ -29,18 +22,11 @@ Due to a compatibility issue with Webpack, Node.js 18.15.0 does not work with Ju
 :::
 
 ## Development install
-
-First, install the Hatch CLI, which installs the Hatchling build backend automatically.
-
-```
-pip install hatch
-```
-
-Then, enter the default hatch environment, which automatically installs all dependencies and executes development setup when entering for the first time. This command must be run from the root of the monorepo (`<jupyter-ai-top>`).
+This command must be run from the root of the monorepo (`<jupyter-ai-top>`).
 
 ```
 cd <jupyter-ai-top>
-hatch shell
+./scripts/install.sh
```

Comment:
There needs to be a preceding paragraph that tells developers to create a new Conda environment w/ Python 3.10.

Reply:
Python 3.11 incompatibility should be highlighted in the user and contributor docs with a warning banner.

Reply:
Example admonition:

```
:::{important}
:name: python-3-11-not-supported
Because the version of Ray that Jupyter AI uses is not compatible
with Python 3.11, please use a supported version of Python up to
and including Python 3.10.
:::
```